Doxyfile

```diff
@@ -38,7 +38,7 @@ PROJECT_NAME           = "Azure SDK for Embedded C"
 # could be handy for archiving the generated documentation or if some version
 # control system is used.
 
-PROJECT_NUMBER         = "1.0.0-preview.1"
+PROJECT_NUMBER         = "1.0.0-preview.2"
```

Comment:
@danieljurek I just want to make sure this should be updated here. I would assume it would?

Reply:
Your change is correct. Eventually we'll have automation handle pulling that in from `az_version.h`